package-res/resources/web/pentaho/visual/base/events/DidUpdate.js

```diff
@@ -18,20 +18,39 @@ define([
 ], function(Event) {
   "use strict";
 
-
+  /**
+   * @name DidUpdate
+   * @memberOf pentaho.visual.base.events
+   * @class
+   * @extends pentaho.lang.Event
+   *
+   * @classDesc This event is emitted when updating the view with
+   * {@link pentaho.visual.base.Model#update|Model#update} occurs without any failures.
```

Comment:
{@link pentaho.visual.base.Model#update|Model#update} and it occurs without any failures.